_posts/2021-08-23-gsoc-2021-project.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: post
+title: Google Summer of Code 2021 - Dask Project
+author: Freyam Mehta
+theme: twitter
+---
+
+{% include JB/setup %}
+
+This blog covers the work done by me during the Google Summer of Code 2021. Dask took part in the program under the NumFOCUS umbrella organization.
+
+Google Summer of Code is a global program focused on bringing more student developers into open source software development. Students work with an open source organization on a 10 week programming project during their break from school.
+
+## Contents
+
+- [Visualizing the Performance Characteristics of Computations](#visualizing-the-performance-characteristics)
+- [Graphical Representation of Task Graphs](#graphical-representation)
+- [HTML Representation of Task Graphs](#html-representation)
+
+## Visualizing the Performance Characteristics of Computations
+
+## Graphical Representation of Task Graphs
+
+I worked on enhancing the graphviz output of the task graphs. The Graphviz library boasts a great set of attributes which can be modifified to create a more visually appealing output. I worked on making them more illustrative, engaging, and informative.
+
+### Fixing calling `.visualize()` with `filename=None` [#7740](https://github.com/dask/dask/pull/7740)
```

Comment:
I'd like this bugfix to go at the end of the first section, since Dask users will be more interested to hear about new features first.

Reply:
Done